src/Hub.cpp

```diff
@@ -233,6 +236,10 @@ sc2::UNIT_TYPEID Hub::GetCurrentWorkerType() const {
     return m_current_worker_type;
 }
 
+sc2::UNIT_TYPEID Hub::GetCurrentTownHallType() const {
```

Comment:
Unused?

Reply:
Using now in an auto-expand check in Miner.